tests/functional/test_plugins.py

```diff
@@ -21,6 +21,7 @@
 logger = get_logger()
 
 
+@pytest.mark.flaky(max_runs=5)
```

Comment:
do we know what specifically makes this flaky? would make a good note/comment

Reply:
unfortunately no. I think there is a bug somewhere in chalk as the timeout is very generious now however so far Im unable to replicate it locally but it does happen with some frequency in CI which is rather annoying